salt/orch/update.sls

```diff
@@ -0,0 +1,121 @@
+# Generic Updates
+update_pillar:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_pillar
```

Comment:
It's good to be safe by doing it (nothing wrong would happen for doing it), but I don't think it's necesary to `refresh_grains`, upon start, `salt-minion` is the very first thing it will do.